examples/otel-logs-with-sourcetypes-splunk/README.md

```diff
@@ -0,0 +1,18 @@
+# Splunk HEC Example
+
+This example showcases how the collector can collect data from files and send it to Splunk Enterprise, associating with each source a different sourcetype.
+
+The example runs as a Docker Compose deployment. The collector can be configured to send logs to Splunk Enterprise.
```

Comment:
Would it be worthwhile to add a little explanation about what this example does? e.g. it creates three pipelines, each with its own filelog receiver and resource processor. Each resource processor sets a `com.splunk.sourcetype` resource attribute to a different value...

Reply:
copied verbatim and finished the sentence, thanks!